x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/support/DLSRoleQueryValidator.java

```diff
@@ -72,26 +74,41 @@ public static void validateQueryField(RoleDescriptor.IndicesPrivileges[] indices
         }
     }
 
-    private static boolean isTemplateQuery(BytesReference query, NamedXContentRegistry xContentRegistry) throws IOException {
+    public static boolean isTemplateQuery(BytesReference query, NamedXContentRegistry xContentRegistry) throws IOException {
```

Comment:
Why `public`?

Reply:
This is a leftover change and should be reverted.